npm/ng-packs/packages/account/src/lib/guards/authentication-flow.guard.ts

```diff
@@ -12,3 +15,12 @@ export class AuthenticationFlowGuard implements IAbpGuard {
     return false;
   }
 }
+
+export const AuthenticationFlowGuardFn = () => {
```

Comment:
why arrow function ?

Reply:
i have followed this syntax because if we dont use arrow functions it forces us to run function as follow
```
canActivate: [AuthGuardFn()]
```

Reply:
we can use function, instead of arrow fn

Reply:
what's the benifit for not use arrow function?
I saw angular doc also use arrow function: https://angular.io/api/router/CanActivateFn#description